lib/Toolkit.js

```diff
@@ -1633,14 +1638,21 @@ class Toolkit {
 
   createUserSpace(name, lib, attr, size, auth, desc, cb) {
     const pgm = new ProgramCall('QUSCRTUS', { lib: 'QSYS' });
-    pgm.addParam([[name, '10A'], [lib === '' ? '*CURLIB' : lib, '10A']]);
-    pgm.addParam(attr === '' ? 'LOG' : attr, '11A');
-    pgm.addParam(size === '' ? 50 : size, '10i0');
-    pgm.addParam(0, '1A');
-    pgm.addParam(auth === '' ? '*USE' : auth, '10A');
-    pgm.addParam(desc, '50A');
-    pgm.addParam('*NO', '10A');
-    pgm.addParam(this.errno, { io: 'both', len: 'rec2' });
+    const qualifiedName = {
+      type: 'ds',
+      fields: [
+        { type: '10A', value: name },
+        { type: '10A', value: lib === '' ? '*CURLIB' : lib },
+      ],
+    };
```

Comment:
Looks like this would be good to pull out of this function so it can be reused, just like errno is above.